cli/lib/dinstaller_cli/commands/user.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# Copyright (c) [2022] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "thor"
+require "dinstaller_cli/clients/users"
+
+module DInstallerCli
+  module Commands
+    # Subcommand to manage first user settings
+    class User < Thor
+      desc "set <name>", "Configure the user that will be created during the installation"
+      option :fullname, banner: "<full-name>", desc: "Set the user's full name"
+      option :password, banner: "<plain-password>", desc: "Set the user's password"
+      option :autologin, type: :boolean, default: false,
+        desc: "Enable/disable user autologin (disabled by default)"
+      def set(name)
+        client.create_first_user(name,
+          fullname:  options[:fullname],
+          password:  options[:password],
+          autologin: options[:autologin])
+      end
+
+      desc "show", "Show the user configuration"
+      def show
+        user = client.first_user
+
+        return if user[1].empty?
+
+        puts "Full Name: #{user[0]}"
+        puts "Name: #{user[1]}"
+        puts "Autologin: #{user[2] ? "yes" : "no"}"
+        puts "Password: <secret>"
```

Comment:
NP: Is there any chance to have a user set without password? I do not think so, but if it is the case this output could be improved.

Reply:
In general, the commands, options, etc have to be revisited. Anyway, for the user password we have no info from our D-Bus API. I would have to extend it.